recipe/conda_build_config.yaml

```diff
@@ -1,2 +1,8 @@
 arm_variant_type: # [aarch64]
   - sbsa          # [aarch64]
+  # arm-variant * tegra requires glibc version which is currently not available.
+  # binary glibc 2.29.0 <= recipe glibc 2.28 $PREFIX/bin/nvcc
```

Comment:
Where did you get the requirement for glibc v2.29.0? CUDA 12.8.1 tegra seems to be ok with glibc v 2.35: https://docs.nvidia.com/cuda/archive/12.8.1/cuda-installation-guide-linux/index.html#id47

v12.9 is supported on ubuntu 22.04 which has glibc v2.35 as well: https://docs.nvidia.com/cuda/archive/12.9.0/cuda-installation-guide-linux/index.html

so we should be fine to build with our builders?